src/components/views/rooms/MessageComposerInput.js

```diff
@@ -523,13 +523,27 @@ export default class MessageComposerInput extends React.Component {
         let sendTextFn = this.client.sendTextMessage;
 
         if (contentText.startsWith('/me')) {
```

Comment:
This could probably be shortened to something like:
```
if (contentText.startsWith('/me') || contentText.startsWith('/my') {
  contentText = contentText.startsWith("/me ") ? contentText.substring(4) : contentText.substring(3);
  // ...
}
```